addons/website_sale/views/templates.xml

```diff
@@ -1032,7 +1032,8 @@
                                                 <i class="fa fa-minus"></i>
                                             </a>
                                         </div>
-                                        <input type="text" class="js_quantity form-control quantity" t-att-data-line-id="line.id" t-att-data-product-id="line.product_id.id" t-att-value="int(line.product_uom_qty) == line.product_uom_qty and int(line.product_uom_qty) or line.product_uom_qty" />
+                                        <t t-set="rounded_uom_qty" t-value="round(line.product_uom_qty,env['decimal.precision'].precision_get('Product Unit of Measure'))"/>
+                                        <input type="text" class="js_quantity form-control quantity" t-att-data-line-id="line.id" t-att-data-product-id="line.product_id.id" t-att-value="int(rounded_uom_qty) == rounded_uom_qty and int(rounded_uom_qty) or rounded_uom_qty" />
```

Comment:
Considering the client is in 17, only complains about the cart notification, and that this solution requires a module update to fix the issue, I don't think it's worth fixing that in 15.
Why don't you fix it only in 17.0, in `_get_cart_notification_information` ? (the issue doesn't happen in the cart template itself in 17)

Reply:
Thanks for the review! I just checked vestion 17.0 and apparently the issue still persist in the cart template. What do you think we should do?